test/phettberg/document.dom.test.ts

```diff
@@ -0,0 +1,39 @@
+import {expect} from 'chai';
+import {BeGone} from '../../src/gendersprachekorrigieren';
+import {replacementTestStrings} from "./testdata";
+import {createParagraph, prepareDocument} from "../domtest-util";
+
+// import './testdata-fehler'
```

Comment:
Dieser Test benutzt die gleichen Testfälle wie `gendersprachekorrigieren.test.ts` (nur als array in `testdata.ts` deklariert), benutzt aber ein "echtes" dom-Document und testet damit auch `beGone.entferneInitial();`.

Die Datei `./testdata-fehler` enthält **neue** Tests, die aktuell noch fehlschlagen. 
Die sind aktuell inaktiv, kann man mit `import './testdata-fehler'` hier mit reinnehmen (dann schlägt aber der test fehl)